src/components/Advertisement/index.js

```diff
@@ -1,8 +1,10 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
 import FmdGoodIcon from '@mui/icons-material/FmdGood';
-import { Paper, Box, Typography } from '@mui/material';
+import { Paper, Box, Typography, Collapse } from '@mui/material';
 import Rating from 'components/Rating';
+import Button from '../Button';
 import Icon from '../Icon';
 import PriceRange from '../PriceRange';
```

Comment:
Zmieńmy ścieżki na absolutne, czyli:
`import Button from 'components/Button';`
i to samo z Icon i PriceRange.
Z relatywnymi potem może być problem jakby chcieć robić jakiś refactoring